services/api/src/routes/users.js

```diff
@@ -52,6 +54,20 @@ router
       };
     }
   )
+  .get(
+    '/:userId/token',
+    requirePermissions({ endpoint: 'users', permission: 'write', scope: 'global' }),
+    async (ctx) => {
+      const { user } = ctx.state;
+      if (!user.authTokenId) {
+        user.authTokenId = generateTokenId();
+        await user.save();
+      }
+      ctx.body = {
+        data: { token: createAuthToken(user.id, user.authTokenId, '120m') },
```

Comment:
Before creating auth token, need to check if target user permissions are HIGHER than authUser. If HIGHER we should not ALLOW ESCALATION

Reply:
(only allow `viewer`)